src/main/java/org/kitteh/irc/client/library/Client.java

```diff
@@ -321,6 +331,83 @@
     }
 
     /**
+     * Proides commands.
+     */
+    interface Commands {
+        /**
+         * Provides a new AWAY command.
+         *
+         * @return new away command
+         */
+        @Nonnull
+        AwayCommand away();
+
+        /**
+         * Provides a new CAP REQ command.
+         *
+         * @return new capability request command
+         */
+        @Nonnull
+        CapabilityRequestCommand capabilityRequest();
+
+        /**
+         * Provides a new MODE command.
+         *
+         * @return new mode command
+         */
+        @Nonnull
+        ChannelModeCommand mode(@Nonnull Channel channel);
+
+        /**
+         * Provides a new KICK command.
+         *
+         * @return new kick command
+         */
+        @Nonnull
+        KickCommand kick(@Nonnull Channel channel);
+
+        /**
+         * Provides a new MONITOR command.
+         *
+         * @return new monitor command
+         */
+        @Nonnull
+        MonitorCommand monitor();
+
+        /**
+         * Provides a new OPER command.
+         *
+         * @return new oper command
+         */
+        @Nonnull
+        OperCommand oper();
+
+        /**
+         * Provides a new TOPIC command.
+         *
+         * @return new topic command
+         */
+        @Nonnull
+        TopicCommand topic(@Nonnull Channel channel);
+
+        /**
+         * Provides a new WALLOPS command.
+         *
+         * @return new wallops command
+         */
+        @Nonnull
+        WallopsCommand wallops();
```

Comment:
I'm not convinced this belongs here either. It already expects the condition of being an oper.

Maybe a separate OperClient that extends Client is a nice idea?